.travis.yml

```diff
@@ -8,10 +8,6 @@ addons:
       - oracle-java9-installer
 
 before_install:
-  - wget https://archive.apache.org/dist/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.zip
```

Comment:
I had added this for stability especially for the newer JDK versions. I would keep this because Travis upgrades the stock maven really slowly, and we will have to add similar steps in the future.
This does speed up the build and I think saves 2-3 minutes, however, longer term I think we should stick to the custom maven version.

Reply:
The script download and unzips maven 3.5.0. Deleting that brings us forward
to 3.5.2.

Reply:
Even if we keep it, it should probably have a link to https://github.com/travis-ci/enterprise-installation/issues/14 for credit